docs/DeveloperGuide.md

```diff
@@ -890,6 +1073,47 @@ testers are expected to do more *exploratory* testing.
     3. If the data is beyond repair, delete the entire `docs` folder and the `AddressBook.json` file to start afresh.
 
 
+---
+
+## Appendix: Efforts
+
+WedLinker is a project adapted from the original [AddressBook3 (AB3)](https://se-education.org/addressbook-level3/) customized specifically for wedding planners by our team of five.
+Throughout this project, we had split the work evenly and adhering to the principles and concepts that were taught by the
```

Comment:
Maybe could reword to "Throughout this project, we split the work evenly and adhered to the principles and concepts that were taught by the ..."